ambry-admin/src/main/java/com.github.ambry.admin/AdminBlobStorageService.java

```diff
@@ -356,9 +357,10 @@ public void onCompletion(String result, Exception exception) {
               RestUtils.SubResource subresource = RestUtils.getBlobSubResource(restRequest);
```

Comment:
Ah, the reason it was done this way was because a `GetCallback` is not passed in for `/replicas` requests.